src/dicomweb_client/api.py

```diff
@@ -1057,6 +1057,22 @@ def _http_get_multipart_application_dicom(
             headers=headers,
             stream=stream
         )
+        # The response of the Retrieve Instance transaction is supposed to
+        # contain a message body with Content-Type "multipart/related", even
+        # if it only contains a single part. However, some origin servers
+        # violate the standard and send the part non-encapsulated.
+        # Unfortunately, an error was introduced into the standard via
+        # Supplement 183 as part of re-documentation efforts, which stated that
+        # this behavior was allowed. We will support this behavior at least
+        # until the standard is fixed via a Correction Proposal 2040.
+        if response.headers['Content-Type'].startswith('application/dicom'):
+            logger.error(
+                'message sent by origin server in response to retrieve '
+                'instance request was not compliant with the standard, '
+                'message body shall have "multipart/related" Content-Type'
+            )
```

Comment:
This is likely to be quite noisy for an application using the client. An alternative to consider is the [`warnings`](https://docs.python.org/3/library/warnings.html) package which allows for more fine-grained filtering.

Reply:
In this case, I would argue the noisier the better

Reply:
Well, I suppose your goal is to get this noticed so that the issue gets fixed, but you'll be yelling at the user of dicomweb-client who probably has no control over the server and is just going to be stuck with it, so you'll only be annoying them.

Plus this really isn't an `error` condition since the transaction succeeds.  Logging this as an error could cause some tests to fail.  It really should be a one-time warning.

How about adding a flag to pass when creating the client to silence the error?

You can even vent your rage by name-shaming them with an option like `accept_obviously_incorrect_content_type` and a second option `i_agree_to_complain_loudly_to_server_maintainer` which must be set to `True`.

Reply:
In many cases, a client application will be using this API to fetch studies/series. Given the lack of compliance is due to the server, the client will have little in the way of means to correct this error. Providing the client application a means to control this level of verbosity seems reasonable. Otherwise, actual errors in the client application will be drowned out in the noise.

Reply:
The functionality @pieper describes above is one of the most common use cases of `warnings`. See [here](https://docs.python.org/3/library/warnings.html#the-warnings-filter) for filter settings.

Reply:
Addressed with 01836807a911f077ca0049bd823345448619a7c2